brianlong/www.validators.app
# 186133526 stake accounts api

#### What's this PR do?
- create API for stake accounts explorer

#### How should this be manually tested?
- run tests
- request endpoint using various parameters as written in documentation
- check the docs

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186133526)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
